data_dir/c-s_lwt_basic_multidc.yaml

```diff
@@ -0,0 +1,79 @@
+# LWT test: create and update data using LWT.
```

Comment:
Why do you need duplicate this user profile? What is the difference?

Reply:
replication = {'class': 'NetworkTopologyStrategy', 'eu-west1scylla_node_west': 3, 'us-west2scylla_node_west': 3, 'us-east1scylla_node_east': 3};

Reply:
If we do that, maybe worth to cover more things?
Other things?
Maybe more column types, different LWT conditions, etc?

Reply:
@juliayakovlev WDTY?

Reply:
@juliayakovlev ?

Reply:
There is another user profile ```data_dir/c-s_lwt_big_data.yaml``` that has different column types. @amoskong  can use it instead of basic user profile

Reply:
BTW, in a recent PR from me (#2129), I introduced a method to "_pre_create_schema".
You can add in the test yaml the "Create keyspace" command.
However, I don't know what happens when running the user profile and the keyspace is already exist, but I guess it should just work.

Reply:
> There is another user profile `data_dir/c-s_lwt_big_data.yaml` that has different column types. @amoskong can use it instead of basic user profile

I will try with big_data profile.